<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SolveNoDataDialog</class>
 <widget class="QDialog" name="SolveNoDataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1424</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Solve NoData issues</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,1,0,0,0,0">
   <item row="0" column="0">
    <widget class="QLabel" name="labelLayers">
     <property name="text">
      <string>Select raster layers:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="6,1,6">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QFrame" name="frameSourceTree">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <widget class="QTreeWidget" name="treeAvailable">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>641</width>
          <height>451</height>
         </rect>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="headerHidden">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_buttons">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>180</y>
          <width>106</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
         <item>
          <widget class="QPushButton" name="btnAdd">
           <property name="text">
            <string>Add →</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRemove">
           <property name="text">
            <string>← Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_list_selected">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <widget class="QTableWidget" name="tblSelected">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>613</width>
          <height>581</height>
         </rect>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::DoubleClicked</set>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <property name="horizontalHeaderLabels" stdset="0">
         <stringlist>
          <string>Layer</string>
          <string>Short Name</string>
         </stringlist>
        </property>
        <column/>
        <column/>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="layoutNoData">
     <item>
      <widget class="QLabel" name="labelNoData">
       <property name="text">
        <string>New NoData value:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinNoData">
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="minimum">
        <double>-1000000000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000000000.000000000000000</double>
       </property>
       <property name="value">
        <double>-9999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="layoutFolder">
     <item>
      <widget class="QLabel" name="labelFolder">
       <property name="text">
        <string>Output folder:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtFolder"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnBrowse">
       <property name="text">
        <string>Browse…</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelPrefix">
       <property name="text">
        <string>Prefix:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtPrefix">
       <property name="placeholderText">
        <string>e.g. nodata_</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSuffix">
       <property name="text">
        <string>Suffix:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtSuffix">
       <property name="placeholderText">
        <string>e.g. _clean</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="layoutOptions">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QCheckBox" name="chkAdd">
       <property name="text">
        <string>Add to project</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkOverwrite">
       <property name="text">
        <string>Overwrite existing</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
